services/web/client/source/resource/osparc/ti-solutions.svg

```diff
@@ -1,87 +1,14 @@
 <svg width="250" height="150" xmlns="http://www.w3.org/2000/svg">
- <style type="text/css">.st0{display:none;}
-	.st1{display:inline;fill:#FFFFFF;}
-	.st2{display:inline;opacity:0.4;fill:url(#SVGID_1_);}
-	.st3{display:inline;opacity:0.4;fill:url(#SVGID_2_);}
-	.st4{display:inline;fill:url(#SVGID_3_);}
-	.st5{display:none;opacity:0.8;fill:url(#SVGID_4_);}
-	.st6{display:none;opacity:0.8;fill:url(#SVGID_5_);}
-	.st7{display:none;fill:url(#SVGID_6_);}
-	.st8{fill:#FFFFFF;}
-	.st9{display:none;fill:url(#SVGID_7_);}
-	.st10{display:none;fill:#3ABFEF;}
-	.st11{fill:#0084B4;}
-	.st12{display:none;fill:#FFFFFF;}
-	.st13{display:none;fill:#FFFFFF;}
-	.st14{font-family:'FrutigerLTStd-Light';}
-	.st15{font-size:36px;}
-	.st16{font-family:'Roboto-Regular';}</style>
-
+ <!-- Created with Method Draw - http://github.com/duopixel/Method-Draw/ -->
  <g>
   <title>background</title>
-  <rect x="-1" y="-1" width="252" height="152" id="canvas_background" fill="none"/>
+  <rect fill="#fff" id="canvas_background" height="152" width="252" y="-1" x="-1"/>
+  <g display="none" overflow="visible" y="0" x="0" height="100%" width="100%" id="canvasGrid">
```

Comment:
so you leave it white! OK.
I do not think you even need to keep the old one... actually the version control already keeps a copy of it